node.js_mongodb/SingleUpload.js

```diff
@@ -0,0 +1,24 @@
+const DBHelper=require('./DB_helper');
+const readline = require('readline');
+PromptAndUpload();
+
+async function PromptAndUpload() {
+    const rl = readline.createInterface({
+        input: process.stdin,
+        output: process.stdout
+      });
+      
+      rl.question('File name: ', async (jsonFileName) => {
+        rl.question('Collection name: ', async (collectionName) => {
+            console.log("Overwriting data..");
+            let db="rsquared_zilla_upazilla_db";
+            var all_data = require('./data/'+jsonFileName);
+            let deleteExisting= await DBHelper.deleteData(db,collectionName,{});
```

Comment:
Will it throw an error if collection does not exist? Can we make it more defensive so that it tries to delete only if it exists?